.github/workflows/keadm_compatibility.yaml

```diff
@@ -0,0 +1,140 @@
+        run: |
+          sudo apt-get install -y jq
+
+  image-prepare:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    name: Prepare kubeedge/build-tools image
+    steps:
+      - name: Pull kubeedge/build-tools image
+        run: |
+          docker pull kubeedge/build-tools:1.20.10-ke1
+          mkdir -p /home/runner/build-tools/
+          docker save kubeedge/build-tools:1.20.10-ke1 > /home/runner/build-tools/build-tools.tar
+
+      - name: Temporarily save kubeedge/build-tools image
+        uses: actions/upload-artifact@v3
+        with:
+          name: build-tools-docker-artifact
+          path: /home/runner/build-tools
+
+  build:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    name: Multiple build
+    needs: image-prepare
+    steps:
+      - uses: actions/cache@v3
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Retrieve saved kubeedge/build-tools image
+        uses: actions/download-artifact@v3
+        with:
+          name: build-tools-docker-artifact
+          path: /home/runner/build-tools
+
+      - name: docker load kubeedge/build-tools image
+        run: |
+          docker load < /home/runner/build-tools/build-tools.tar
+
+      - run: make
+
+      - run: make smallbuild
+
+      - run: make crossbuild
+
+      - run: make crossbuild ARM_VERSION=GOARM7
+
+      - run: make crossbuild WHAT=cloudcore ARM_VERSION=GOARM8
+
+  keadm_compatibility_e2e_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        version:
+          - v1.15.1
+          - v1.14.4
+    timeout-minutes: 90
+    name: Keadm compatibility e2e test
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.20.x
+
+      - uses: actions/cache@v3
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - name: Install dependences
+        run: |
+          command -v ginkgo || go install github.com/onsi/ginkgo/v2/ginkgo@${{ env.GINKGO_VERSION }}
+          go install sigs.k8s.io/kind@v0.19.0
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.27.7/bin/linux/amd64/kubectl && sudo install kubectl /usr/local/bin/kubectl
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.4.0
+
+      - name: enable cri config in containerd service
+        run: |
+          containerd config default | sudo tee /etc/containerd/config.toml && sudo systemctl restart containerd.service
+
+      - name: cleanup images
+        run: docker system prune -a -f
+
+      - run: |
+          export EDGECORE_IMAGE=${{ matrix.version }}
```

Comment:
Why is the version exported as EDGECORE_IMAGE?
EDGECORE_VERSION is a better choice.

Reply:
done